### PR TITLE
docs(aio): several fix for ngmodule guides

### PR DESCRIPTION
**Pull request by trotyl**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/guide/entry-components.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ In fact, many libraries declare and export components you'll never use.
 For example, a material design library will export all components because it doesn’t know which ones the you will use. However, it is unlikely that the you will use them all.
 For the ones you don't reference, the tree shaker drops these components from the final code package.
 
-If a component isn't an _entry component_ or isn't found in a template,
+If a component isn't an _entry component_ and isn't found in a template,
 The tree shaker will throw it away. So, it's best to add only the components that are truly entry components to help keep your app
```
Comment (alfaproject) [inline aio/content/guide/entry-components.md]: Lowercase `The`?

**Comment by gkalpak** [comment]
@trotyl, could you please rebase this and fix the conflicts? ❤️

File: aio/content/guide/ngmodule-vs-jsmodule.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ JavaScript modules help you namespace, preventing accidental global variables.
 ## NgModules
 
 <!-- KW-- perMisko: let's discuss. This does not answer the question why it is different. Also, last sentence is confusing.-->
-NgModules are classes decorated with `@NgModule`. The `@NgModule` decorator’s `imports` array tells Angular what other NgModules the current module needs. The modules in the imports array are different than JavaScript modules because they are NgModules rather than regular JavaScript modules. Classes with an `@NgModule` decorator are by convention kept in their own files, but what makes them an `NgModule` isn’t being in their own file, like JavaScript modules; it’s the presence of `@NgModule` and its metadata.
+NgModules are classes decorated with `@NgModule`. The `@NgModule` decorator’s `imports` array tells Angular what other NgModules the current module needs. The modules in the `imports` array are different than JavaScript modules because they are NgModules rather than regular JavaScript modules. Classes with an `@NgModule` decorator are by convention kept in their own files, but what makes them an `NgModule` isn’t being in their own file, like JavaScript modules; it’s the presence of `@NgModule` and its metadata.
```
Comment (trotyl) [inline aio/content/guide/ngmodule-vs-jsmodule.md]: ```The modules in the `imports` array``` add the ` around imports here, the diff seems not work

**Comment by trotyl** [comment]
@gkalpak Done~

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**